django/urls/resolvers.py

```diff
@@ -368,73 +415,58 @@ def _populate(self):
         namespaces = {}
         apps = {}
         language_code = get_language()
-        regex, converters = self.pattern.compile()
-        # We need to store these two values *before* inspecting url_patterns,
-        # because there might be cyclic dependencies.
```

Comment:
Is the problem with cyclic dependencies solved? I *think* so because it has been moved to `pattern` instead.